docs/generator/buildyaml.sh

```diff
@@ -174,6 +174,7 @@ navpart 3 collectors/freebsd.plugin
 navpart 3 collectors/macos.plugin
 
 navpart 2 collectors/plugins.d "" "External plugins"
+navpart 3 collectors/go.d.plugin "" "go modules" 3
```

Comment:
`Go modules`

Reply:
This is very misleading as most common usage of phrase `go modules` refers to https://github.com/golang/go/wiki/Modules

Reply:
Can we finally agree on definitions?

https://github.com/netdata/netdata/issues/5311#issuecomment-460247178

Reply:
Every time somebody mention `module` there is a guy with a link to go modules :smile:

Reply:
The decision on #4394 is to proceed for now exactly as @ilyam8 mentioned in the comment he referenced. Just because Go uses "module" for something else doesn't mean we HAVE to change our terminology.

Reply:
changed go to Go

Reply:
@paulfantom is right btw, they are not `go modules`, they are `go.d.plugin modules` in fact.

Reply:
I have to change the other three as well if we do that. 
i.e.
python.d.plugin modules
node.d.plugin modules
charts.d.plugin modules

I'd rather just remove the word `modules` completely from the menu.

Reply:
> I'd rather just remove the word `modules` completely from the menu.

:+1: 

`language modules` => `language.d.plugin` and move `modules` section inside

Reply:
We reviewed the latest version with @ilyam8